### DISCUSSION TITLE
Subcategories

### DISCUSSION THREAD

**Discussion by finnjosten** [Paymenter/Paymenter#191] [Ideas]
This can work also instead of -> https://github.com/Paymenter/Paymenter/discussions/190

For example i have a Minecraft category, would be nice if you can make a sub categories for Java or Bedrock hosting. (this can also work with more config options for a product).

This will make it so you don't have to make 1 big category and sub divide each product with name or so. Or having to make like multiple main categories for each thing.

**Comment by purejosh** [comment]
This is currently possible, is it not? I already have mine set up similarly, where I have a Minecraft category, and a Vanilla category under the Minecraft one.

**Reply by finnjosten** [reply]
I dont want to be annoying, but maybe read how long ago this was suggested?

In the mean time it has been added yes

**Reply by purejosh** [reply]
Not annoying to me at all. But these should probably be closed out as they're added to the project.

**Reply by finnjosten** [reply]
Tbh not really my problem but yes they should. I will close it for ya rq
